flow/platforms/ihp-sg13g2/lib/sg13g2_stdcell_typ_1p20V_25C.lib

```diff
@@ -58,114 +58,114 @@ library (sg13g2_stdcell_typ_1p20V_25C) {
     temperature : 25;
     voltage : 1.2;
   }
- wire_load("Zero") {
-    capacitance: 0.0001;
-    resistance: 0.0001;
-    area : 0.01; 
-    slope : 1.0;
-    fanout_length(1, 0.01);
-    fanout_length(5, 0.01);
-    fanout_length(20, 0.01);
-}
-wire_load("0_1k") {
-    capacitance: 0.0001;
-    resistance: 0.0035;
-    area : 0.36; 
-    slope: 10.83;  
-    fanout_length(1, 6.43);
-    fanout_length(5, 38.56);
-    fanout_length(20, 146.8);
-}
-wire_load("0_5k") {
-    capacitance: 0.0001;
-    resistance: 0.0042;
-    area : 0.36; 
-    slope: 10.99; 
-    fanout_length(1, 7.89);
-    fanout_length(5, 47.32);
-    fanout_length(20, 169.4);
-}
-wire_load("1k") {
-    capacitance: 0.0002;
-    resistance: 0.0053;
-    area : 0.36; 
-    slope: 10.9; 
-    fanout_length(1, 9.93);
-    fanout_length(5, 59.59);
-    fanout_length(20, 215.6);
-}
-wire_load("2k") {
-    capacitance: 0.0002;
-    resistance: 0.0063;
-    area : 0.36; 
-    slope: 16.87;  
-    fanout_length(1, 11.68);
-    fanout_length(5, 70.11);
-    fanout_length(20, 239.5);
-}
-wire_load("5k") {
-    capacitance: 0.0003;
-    resistance: 0.0072;
-    area : 0.36; 
-    slope: 14.58;  
-    fanout_length(1, 13.44);
-    fanout_length(5, 80.62);
-    fanout_length(20, 287.1);
-}
-wire_load("10k") {
-    capacitance: 0.0003;
-    resistance: 0.008;
-    area : 0.37; 
-    slope: 18.94;  
-    fanout_length(1, 14.9);
-    fanout_length(5, 89.39);
-    fanout_length(20, 381.6);
-}
-wire_load("30k") {
-    capacitance: 0.0003;
-    resistance: 0.0094;
-    area : 0.37; 
-    slope: 24.61;  
-    fanout_length(1, 17.53);
-    fanout_length(5, 105.2);
-    fanout_length(20, 515.5);
-}
-wire_load("50k") {
-    capacitance: 0.0005;
-    resistance: 0.0152;
-    area : 0.37; 
-    slope: 31.81;  
-    fanout_length(1, 28.33);
-    fanout_length(5, 170);
-    fanout_length(20, 607.6);
-}
-wire_load("100k") {
-    capacitance: 0.0009;
-    resistance: 0.0265;
+  wire_load (Zero) {
```

Comment:
Any reason this isn't quoted but others are?

Reply:
@rovinski thank you for finding this, I will try to fix it ASAP

Reply:
I think either is allowed in Liberty.   100k might be confusing without the quotes but I haven't tried it.

Reply:
@KrzysztofHerman It's not an error so much as a consistency question